### DISCUSSION TITLE
Is VADIterator and get_speech_timestamps available in onnx model?

### DISCUSSION THREAD

**Discussion by shrestha-pranita** [snakers4/silero-vad#237] [Q&A] [answered]
Hi,
I wanted to ask you if the VADIterator and get_speech_timestamps available in onnx model? By far I can only see the results from Decoder.

**Comment by snakers4** [comment, marked as answer]
Hi,

ONNX examples differ only by having an ONNX wrapper:

- https://github.com/snakers4/silero-vad/blob/7c671a75c2f82830a550ab4a182e86d3a2609d70/utils_vad.py#L10

**Reply by shrestha-pranita** [reply]
Got it!! Thanks a lot.
